skcosmo/metrics/reconstruction_measures.py

```diff
@@ -0,0 +1,60 @@
+from sklearn.pipeline import make_pipeline, Pipeline
+from sklearn.linear_model import RidgeCV
+from sklearn.model_selection import train_test_split
+
+from ..preprocessing import NormalizeScaler
+from ..linear_model import OrthogonalRegression
+
+import numpy as np
+
+def pointwise_gfre_val_score(features_source, features_target, train_size=0.5, test_size=None, random_state=0x5F3759DF, shuffle=True, n_jobs=None, scaler=NormalizeScaler(), estimator=RidgeCV(alphas=[10**i for i in range(-9,0)]+[0.5, 0.9], cv=2, scoring='neg_root_mean_squared_error')):
```

Comment:
Instead of creating instances for the scaler & estimator in the parameter list, I would set them to None by default, and check in the function if they are None and replace with the defaults we want.

Using mutable objects as default argument values can lead to surprising behaviour:
```python
def foo(bar = []):
    bar.append(3)
    print(bar)

# what does this print?
foo()
foo()
```